st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -211,9 +245,19 @@ def get_system_information():
 
     return system_information
 
+def get_installed_rpm(output_file_path):
```

Comment:
The set of commands to run should be loaded from the yaml file. The yaml file that we contribute back to st2 should not include any commands to run since they don't have this functionality today.

Our yaml file in Plexxi/connect will include "rpm -qa" and likely additional commands to run.